### PR TITLE
Climber inital implementation

### PR DESCRIPTION
**Pull request by aaron97neu**
Should be ready for review. Needs further work once physical climber is built

- [x] Code builds successfully
- [x] Code simulates successfully

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Climber/Climber.java [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        .setMaxVelocity(maxVel)
+        .burnFlash();
+  }
+
+  /**
+   * Extend arm to position given.
+   *
+   * @param pos Desired position in cm from fully retracted position
+   */
+  public void setArmHeight(double pos) {
```
Comment (aaron97neu) [inline src/main/java/frc/robot/subsystems/Climber/Climber.java:52]: @GavinArchon and I figured out that this style of comment (JavaDoc maybe?) can be read by VSCode and is show when calls to this method are hovered over. Extremely helpful and might try to make this a habit for the robot

Comment (aaron97neu) [reply src/main/java/frc/robot/subsystems/Climber/Climber.java:52]: Syntax ref: https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#format

**Comment by aaron97neu** [comment]
Does not simulate due to subsystem config issues atm

File: src/main/java/frc/robot/subsystems/Climber/Climber.java (hunk 1 above)
Comment (KomeiOkamoto12390) [reply src/main/java/frc/robot/subsystems/Climber/Climber.java:52]: You guys overided my javadoc in faceTotag lol

**Comment by KomeiOkamoto12390** [comment]
> Does not simulate due to subsystem config issues atm

It did simulate successfully. Just adding subsystem for simulation on betabot.

**Pull request closed without merging**